distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -1,32 +1,11 @@
 from Standard.Base import all
-import project.Any.Any
-import project.Data.Array.Array
-import project.Data.Index_Sub_Range.Index_Sub_Range
-import project.Data.Json.JS_Object
-import project.Data.Numbers.Integer
-import project.Data.Text.Encoding.Encoding
-import project.Data.Text.Extensions
-import project.Data.Text.Matching_Mode.Matching_Mode
-import project.Data.Text.Text
-import project.Data.Time.Time_Of_Day.Time_Of_Day
-import project.Data.Vector.Vector
-import project.Error.Error
 import project.Errors.Common.Dry_Run_Operation
 import project.Errors.Common.Forbidden_Operation
 import project.Errors.Encoding_Error.Encoding_Error
 import project.Errors.File_Error.File_Error
 import project.Errors.Illegal_Argument.Illegal_Argument
-import project.Errors.Problem_Behavior.Problem_Behavior
-import project.Meta
-import project.Nothing.Nothing
-import project.Panic.Panic
 import project.Runtime.Context
 import project.Runtime.Managed_Resource.Managed_Resource
-import project.System.File.File_Access.File_Access
-import project.System.File.File_Permissions.File_Permissions
-
-from project.Data.Boolean import Boolean, True, False
-from project.System.File_Format import Auto_Detect, File_Format
```

Comment:
How did you find these imports aren't needed, @Akirathan? Do we have _"unused import warning"_?

Reply:
It is not an *unused import*, but rather a *duplicated import*. And yes, we do have a warning for that now. I eliminated a lot of duplicated imports.

Reply:
Please don't remove these ones but remove `from Standard.Base import all`

Reply:
Done in 7dfdd36095bf3a389446a8172d18adb1bb68126a